### PR TITLE
Redirection depend on roles of user

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
### Closes 
- https://github.com/Real-Dev-Squad/website-my/issues/426

### This issue depend on pr:
- https://github.com/Real-Dev-Squad/website-my/pull/435

### What is the change?

- If user choose role **developer** on new-signup form then user will redirect to **www.realdevsquad.com/join**
- If user choose any of these role i.e. **designer, mavens, developers** on new-signup then user will redirect to **www.realdevsquad.com**

### After video

https://github.com/Real-Dev-Squad/website-www/assets/107163260/c30864f3-a572-4ab3-899f-ec3dac7add63

### CODE REVIEW

File: js/goto.js [hunk 1]
```diff
@@ -9,10 +12,21 @@ function redirectUserToPage(page) {
 function redirectionHandler(data) {
   if (data.incompleteUserDetails) {
     redirectUserToPage(SIGNUP_URL);
-  } else if (hasVisitedJoin == 'true' || hasVisitedJoin == null) {
-    redirectUserToPage(`${HOME_URL}/join`);
+  } else if (urlParams.get('dev') === 'true') {
+    if (
+      data.roles.developer == true &&
```
Comment (apurvkhare) [inline js/goto.js:17]: instead of `date.roles.developer == true` just `data.roles.developer` should suffice. Also, for comparison use `===` not `==`.

**Review by apurvkhare — COMMENTED** [review]
use the strict equality comparator wherever you are comparing values

File: js/goto.js (hunk 1 above)
Comment (shubhamsigdar1) [reply js/goto.js]: Thank you @apurvkhare for learning and I am done with changes.

**Comment by apurvkhare** [comment]
> use the strict equality comparator wherever you are comparing values
@shubhamsigdar1 
I still see `==` instead of `===` at 4 places. Please update.

**Review by apurvkhare — APPROVED** [review]

**Review by heyrandhir — COMMENTED** [review]
@shubhamsigdar1 can we have the tests for the changed line updated as well.

**Comment by shubhamsigdar1** [comment]
> @shubhamsigdar1 can we have the tests for the changed line updated as well. Also 1 of the workflow seems to be failing.

Hi @heyrandhir, website-www testing is not setup and this pr is pending from long. The feature is behind feature flag
Please approve now from next time I will account this thing in my deadline.

**Review by heyrandhir — APPROVED** [review]

**Pull request merged**